scripts/config-diff.sh

```
@@ -0,0 +1,491 @@
+    print_subheader "Agents"
+    
+    # Files to exclude from comparison (documentation files)
+    local excluded_files=("README.md" "AGENT_TEMPLATE.md" "AGENT_CATEGORIES.md" "AUDIT_VERIFICATION_PROTOCOL.md")
+    
+    # Get all agent files from both directories
+    local repo_agents=()
+    local user_agents=()
+    
+    if [[ -d "$REPO_AGENTS_DIR" ]]; then
+        while IFS= read -r -d '' file; do
+            local basename=$(basename "$file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)